### PR TITLE
Feat/sof 6875

### PR DESCRIPTION
**Pull request by seankwarren**
This PR includes
- flags on the all property props used to generate RJS Form for user input

### CODE REVIEW

File: schema/properties_directory/workflow/convergence/ionic.json [hunk 1]
```diff
@@ -6,7 +6,8 @@
         "name": {
             "enum": [
                 "convergence_ionic"
-            ]
+            ],
+            "$comment": "isGenerative:true"
```
Comment (azech-hqs) [inline schema/properties_directory/workflow/convergence/ionic.json]: I think we should explain in `README.md` what `isGenerative` means and is used for.

**Review by azech-hqs — COMMENTED** [review]
minor comment about documentation, otherwise looks good 👍

File: schema/properties_directory/workflow/convergence/ionic.json (hunk 1 above)
Comment (seankwarren) [reply schema/properties_directory/workflow/convergence/ionic.json]: section added to `README.md` explaining purpose of flag and linking to example files

**Review by azech-hqs — APPROVED** [review]
looks good 👍

File: README.md [hunk 2]
```diff
@@ -106,6 +106,11 @@ If one wishes to add a new unit functional to that list, please
 - edit the [prototype file](schema/models_directory/pb/qm/dft/dft_unit_functionals_proto.json) and
 - generate a new list of unit functional by running python tests, for example (via `generate_dft_unit_functionals()` from the [esse.functionals](src/py/esse/functionals.py) python module).
 
+### A word on properties
```
Comment (timurbazhirov) [inline README.md]: This title is a bit too vague (I sense this was following @spectre007's "word on functional"), we aren't writing a novel but technical documentation. Let's change this to more informative: "Generative vs Non-generative keys for Property Schemas". And move `### A word on functionals" down below to "## Additional" section or smth like that.

File: schema/properties_directory/elemental/atomic_radius.json [hunk 3]
```diff
@@ -12,7 +12,8 @@
         "name": {
             "enum": [
                 "atomic_radius"
-            ]
+            ],
+            "$comment": "isGenerative:true"
```
Comment (timurbazhirov) [inline schema/properties_directory/elemental/atomic_radius.json]: 1. Do we ever explain what `isGenerative` actually is? We should add some documentation for this, perhaps even to the user documentation and refer to here.
2. We don't need to inject `isGenerative:true` into every `name` key, but instead assume that `name` is the default generative per https://github.com/Exabyte-io/web-app/pull/2270/files#r1303651176.
3.

Comment (seankwarren) [reply schema/properties_directory/elemental/atomic_radius.json]: 1. the flag's purpose/meaning is described in the `README.md`
2. flag removed from all names and included by default.

File: README.md (hunk 2 above)
Comment (seankwarren) [reply README.md]: titles changed, additional sentence added to generative keys description, and example links updated

**Review by timurbazhirov — APPROVED** [review]

**Pull request merged**